nuntium/templates/nuntium/profiles/messages_per_instance.html

```diff
@@ -26,7 +26,7 @@
     {% blocktrans count message_count=writeitinstance.message_set.count %}
     <strong>1</strong> message in your WriteIt
```

Comment:
Ah, Ignore that — Just worked out that I got the plural but not the singular. Yep, should definitely fix that.